sale_restricted_qty/tests/test_sale.py

```diff
@@ -0,0 +1,201 @@
+# Copyright 2019 Akretion
+# Update (Migration) 2022 Ooops - Ashish Hirpara <hello@ashish-hirpara.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import odoo.tests.common as common
+from odoo.exceptions import ValidationError
+from odoo.tests import tagged
+
+
+@tagged("post_install", "-at_install")
+class TestSaleOrderLineMinQty(common.TransactionCase):
+    def setUp(self):
```

Comment:
I would have changed this to classmethod and add this line to improve performances:

`cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))`

Reply:
Done...